src/planning.py

```diff
@@ -101,12 +105,35 @@ def sesame_plan(
             predicates |= new_predicates
 
 
-def task_plan(
+def task_plan_grounding(
```

Comment:
In `sesame_plan`, the grounding is part of the overall timeout. But that's no longer the case here. Should we fix this?

Reply:
hmm. I'll leave it to the user, which in this case would be grammar search. and for now I'm not going to change it because I think adding more timeouts could only hurt us at this point (where things are not yet working). but we can revisit in the future once things are working and we want optimizations